CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - 1 new program
 	1. `animedoro`
 ### Changed
+- Sound playing system updated
+- Python 3.5 dropped
 - `short-break` program duration changed from `10 minutes` to `5 minutes`
 - `long-break` program duration changed from `30 minutes` to `15 minutes`
 - `pomodoro` program updated
```

Comment:
It is crucial to mention the addition of a new dependency. 
- `nava` added to `requirements.txt`

Reply:
fixed in [38f4a1f](https://github.com/sepandhaghighi/mytimer/pull/86/commits/38f4a1f2f12e4618e78b4b81ea4d904d71f8bddf).